specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -6046,6 +6046,9 @@
         },
         "automaticOSUpgradeProperties": {
           "$ref": "#/definitions/AutomaticOSUpgradeProperties"
+        },
+        "hyperVGeneration": {
```

Comment:
>hyperVGeneration [](start = 9, length = 16)

Can you confirm that this is a read-only property?

Reply:
@khan15, could you confirm?